pkg/alertmanager/multitenant.go

```diff
@@ -119,7 +120,7 @@ func (cfg *MultitenantAlertmanagerConfig) RegisterFlags(f *flag.FlagSet) {
 
 	f.BoolVar(&cfg.EnableAPI, "experimental.alertmanager.enable-api", false, "Enable the experimental alertmanager config api.")
 	f.IntVar(&cfg.APIConcurrency, "alertmanager.api-concurrency", 0, "Maximum number of concurrent GET API requests before returning an error.")
-
+	f.DurationVar(&cfg.GCInterval, "alertmanager.gc-interval", 30*time.Minute, "Garbage collection Interval")
```

Comment:
The flag and the configuration name is kind of confusing. Can we make the name and flag description more informative like `alertmanager.alerts-gc-interval`

Reply:
sure, I will modify it.